mitodl/mit-open
# Learning resource drawer design updates

### What are the relevant tickets?

Closes [Learning Resource Drawer #4312](https://github.com/mitodl/hq/issues/4312)

### Description (What does it do?)

Implements the design features and resource run selection for various resource types:

**Courses:** 
https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=2636-61736&m=dev

**Programs:**
https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=2643-66866&m=dev

**Courses / Programs, multiple runs:**
https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=2643-69359&m=dev

**Videos:**
https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=2643-69514&m=dev

### Screenshots (if appropriate):

<img width="450" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/bf32a72d-8429-4721-b02f-d5e8e05efbc8">

-------

<img width="450" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/aa029591-57f8-45dd-925a-4070a0b7491c">

-------

<img width="450" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/de328acc-813a-4f28-90ec-e142ea65fb48">

-------

<img width="300" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/1fd903c9-471e-4eac-a850-3dc14e9a33f2">

### How can this be tested?

Run locally against RC with `yarn watch:rc` and load localhost:8062 or local stack with `docker compose up nginx web watch` and load localhost:8063.

Select learning resources from the "Featured Courses" and "Media" sections on the homepage to open the drawer.

Different resource types have display variations. Verify that courses, programs, podcasts and videos render according to designs.

### Additional Context

- Courses and programs share the default view, with different treatment for podcasts and videos. The remaining resource types learning paths, podcast episodes and video playlists render as default, pending designs.
- Share button is to be added in a subsequent PR.
- "Similar Learning Resources" accordion is to be added in a subsequent PR.
